fuse.py

```diff
@@ -606,6 +685,12 @@ def fuse_exit():
     Flags the native FUSE session as terminated and will cause any running FUSE
     event loops to exit on the next opportunity. (see fuse.c::fuse_exit)
     '''
+    # OpenBSD doesn't have fuse_exit
+    # instead fuse_loop() gracefully catches SIGTERM
+    if _system == "OpenBSD":
```

Comment:
That's the way I [initially implemented](https://github.com/fusepy/fusepy/issues/97) the shutdown, so that should be fine.